L/LLVM/common.jl

```diff
@@ -793,7 +800,8 @@ function configure_extraction(ARGS, LLVM_full_version, name, libLLVM_version=not
     end
 
     dependencies = BinaryBuilder.AbstractDependency[
-        Dependency("Zlib_jll"), # for LLD&LTO
+        Dependency("Zlib_jll"), # for LLD&LTO&debuginfo
+        Dependency("Zstd_jll"), # for debuginfo
```

Comment:
This means also Zstd_jll will have to become an stdlib.

Reply:
You're in luck haha: https://github.com/JuliaLang/julia/pull/58344